src/NzbDrone.Core/Qualities/Quality.cs

```diff
@@ -67,12 +67,12 @@ public override bool Equals(object obj)
         public static Quality HDTV1080p => new Quality(9,  "HDTV-1080p");
         public static Quality RAWHD => new Quality(10, "Raw-HD");
         //public static Quality HDTV480p    { get { return new Quality(11, "HDTV-480p"); } }
```

Comment:
SDTV as a source is hardly seen anymore, adding HDTV 480p seems mostly ceremonious, but could also be confusing, since pretty much everything "SDTV" would become it.

Bluray 480p makes sense, which would include `BDRip` (direct from BR Source to 480p) and `BRRip` (re-encode from a 1080p release). What do we want to name this group? `Disk 480p` maybe?

Also what do we want to do for Remux? There is a separate issue for it, but I'd like to land it as part of this PR if we're going to join group it with BR 1080p/2160p (and people can break the group), otherwise we can add it as a separate quality or would this fall under preferred words? I realize the source is the same, but it feels different enough in quality it may warrant it's own quality.